### ISSUE TITLE
인도 bollard xml, jpg 데이터 수집 

### ISSUE THREAD

**Issue by dyjs-js** [dyjs-js/etg2022WISET#4]
인도 bollard xml, jpg데이터 수집

**Comment by hahory** [comment]
토요일 멘토링 시간에 이슈별로 branch를 만들어 관리하라고 들은대로 이 이슈에 대한 branch를 만들었어요.. branch를 만들 때 sourch branch를 설정해야 하더라구요.. branch를 만들면 sourch branch의 목록도 같이 뜨네요.. 그나마 제 branch(hahory branch)가 xml 추출 관련 코드들이라 제 branch를 source branch로 설정한 것이니 참고하세요.. 어렵네요, 깃허브ㅠㅠㅠㅠㅠㅠㅠ

0828로 시작하는 파일이 bollard 클래스가 있는 이미지만 추출해서 txt파일 생성한 코드입니다.. 제가 처음에 할당받은 2600장에서는 다른 클래스 제외해버리면 500여장의 이미지밖에 남지 않아서 우리팀이 사용한 전체 이미지 20000여장(파일 넘버링으로는 MP_SEL_022800까지)으로 진행한 것이니 참고하세요~=>5701장의 bollard 이미지&txt 만들어짐
이미지가 많아서 코드 셀마다 30초 정도 소요되는 셀들이 있으니 참고하세요..

**Comment by dyjs-js** [comment]
앗 은정님 그래서 제 생각은 브랜치를 만들지 말고 폴더를 만들어서 거기에 업로드하는게 어떨가 싶었거든요.
저희가 아무래도 브랜치 사용이 원활하지않고 일괄적으로 main으로 pull 하는게 좀 부담일거같더라구요. 그때 강사님도 폴더 활동하는것도 좋다고 하셨던거같아서요!

**Comment by hahory** [comment]
@dyjs-js 아~~ main branch의 data폴더 안에 테스트용으로 넣으셨던 것처럼 main branch에 이슈이름으로 폴더를 생성해서 하자는 말이죠?! 훨씬 편할 거 같아요!!!

**Comment by dyjs-js** [comment]
@hahory 네 맞습니다! 구글코랩에서도 업로드가 쉽고 폴더 이용하는게 좋을거같아요. 자세한 방법에 대해서는 제가 정리해서 다시 공유하겠습니다!
